pkg/didcomm/messaging/service/httpmsgsvc.go

```diff
@@ -0,0 +1,53 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package service
```

Comment:
http message service could be its own package.

Reply:
fixed, 
tried to avoid stutter `service.HTTPOverDIDComm` vs 'http.OverDIDComm'